configcatclient/localfiledatasource.py

```diff
@@ -50,8 +58,12 @@ def _reload_file_content(self):
                                 value_type = STRING_VALUE
                             elif isinstance(value, int):
                                 value_type = INT_VALUE
-                            else:
+                            elif isinstance(value, float):
                                 value_type = DOUBLE_VALUE
+                            else:
+                                value_type = UNSUPPORTED_VALUE
+                                self.log.error("Error occurred while parsing the local config file \'%s\':"
+                                               "Unsupported value type: %s." % (self._file_path, type(value)))
```

Comment:
This is a non-agreed log event. Other SDKs don't log such an error message currently. So, I suggest removing this or at least changing it to a debug level event until we standardize the behavior in this case.